torchaudio/transforms.py

```diff
@@ -67,6 +67,12 @@ class Spectrogram(torch.nn.Module):
             This argument is only effective when ``power=None``. It is ignored for
             cases where ``power`` is a number as in those cases, the returned tensor is
             power spectrogram, which is a real-valued tensor.
+
+            Example
+                    >>> specgram = torch.randn(1, 40, 1000)
+                    >>> specgram = torchaudio.transforms.Spectrogram()(specgram)
+                    >>> specgram.shape
+                    torch.Size([1, 40, 201, 6])
```

Comment:
1. This indentation is too much, there should be only 4 spaces instead of 8.
2. Please assign the transform into intermediate variable.
3. Please provide some arguments to `Spectrogram` constructor.
3. The input to `Spectrogram` should be called `waveform`.